### PR TITLE
feat: Add docker Workflow

### PR DESCRIPTION
**Pull request by fabiodmota**
## Description
- Add workflow for creation on docker hub registry for country risk frontend

Fixed for issues:
https://github.com/eclipse-tractusx/vas-country-risk/issues/28

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by alexsilva-CGI — APPROVED** [review]

File: .idea/sonarlint/issuestore/index.pb
Comment (SebastianBezold) [inline .idea/sonarlint/issuestore/index.pb:1]: Is this file added by accident?

File: .github/workflows/docker-hub-build.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+      # https://github.com/peter-evans/dockerhub-description
+      # Important step to push image description to DockerHub 
+      - name: Update Docker Hub description
+        if: github.event_name != 'pull_request'
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          # readme-filepath defaults to toplevel README.md, Only necessary if you have a dedicated file with your 'Notice for docker images'
+          # readme-filepath: path/to/dedicated/notice-for-docker-image.md
```
Comment (SebastianBezold) [inline .github/workflows/docker-hub-build.yml:73]: You add a dedicated `DOCKER_NOTICE.md` with this PR, so you should reference it to be pushed as your DockerHub description. If you do not specify a file, the top-level `README.md` will be used and yours does not contain the base image notice

File: DOCKER_NOTICE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## Notice for Docker image
+
+DockerHub: [https://hub.docker.com/u/tractusx/vas-country-risk](https://hub.docker.com/r/tractusx/vas-country-risk)
```
Comment (SebastianBezold) [inline DOCKER_NOTICE.md:3]: Link results in 404 again

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: .idea/sonarlint/issuestore/index.pb
Comment (fabiodmota) [reply .idea/sonarlint/issuestore/index.pb:1]: Yes from Intelij

File: DOCKER_NOTICE.md (hunk 2 above)
Comment (SebastianBezold) [reply DOCKER_NOTICE.md:3]: image not yet present, therefore cannot resolve

File: .idea/sonarlint/issuestore/index.pb
Comment (SebastianBezold) [reply .idea/sonarlint/issuestore/index.pb:1]: but why would you commit sonarlint issues found by your IDE?

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**